root/usr/lib/systemd/system/lldpad_enable_all.service

```diff
@@ -0,0 +1,10 @@
+[Unit]
+Description=Enable LLDP on all interfaces
+After=lldpad.service
+
+[Service]
+Type=oneshot
+ExecStart=/usr/local/bin/lldptool_enable_all.sh
+
+[Install]
+WantedBy=multi-user.target
```

Comment:
This will not work correctly, systemd will sometimes spawn this *before* network drivers are even loaded. We faced this kind of issues already. See what we do in `22-discovery.ks` already for network manager configuration via udev:

```
cat > /etc/udev/rules.d/81-nm-prepare.rules <<'UDEV'
ACTION=="add", SUBSYSTEM=="net", NAME!="lo", RUN+="/usr/bin/systemd-cat -t nm-prepare /usr/bin/nm-prepare %k"
UDEV
```

Please run the enable script in a similar way for each particular device (one by one). Also rename it to `enable-lldp` without extension.